3-WebApp-multi-APIs/Startup.cs

```diff
@@ -39,8 +38,7 @@ public void ConfigureServices(IServiceCollection services)
                 options.HandleSameSiteCookieCompatibility();
             });
 
-            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
-                .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))
+            services.AddMicrosoftIdentityWebAppAuthentication(Configuration)
```

Comment:
I was under impression the single line is the trend now. Is there any reason to stick to two lines?

Reply:
Where did you get this impression from, @aremo-ms ?
the single line is simpler, but:
1. as soon as customers want to do things which are a bit more involved, they have to use the form with the delegates which is not available with the single line
2. this doesn't allow for multi-authentication-schemes.

We now prefer to use the 2 lines form.

Reply:
I agree that 2 lines allow more flexible configuration. I've got the impression because Readme of all 1-WebApp-OIDC samples, all 4-WebApp-your-AP samples and 5-1 sample. I will revert the change.

Reply:
Thanks @aremo-ms. We'll update the README.md as well (consistently with the code of course)